docs/tutorials/11_quadratic_hamiltonian_and_slater_determinants.ipynb

```diff
@@ -0,0 +1,467 @@
+    "\n",
+    "hamiltonian = QuadraticHamiltonian(\n",
+    "    hermitian_part=hermitian_part,\n",
+    "    antisymmetric_part=antisymmetric_part,\n",
+    "    constant=constant,\n",
+    ")\n",
+    "\n",
+    "# convert it to a FermionicOp and print it\n",
+    "hamiltonian_ferm = hamiltonian.to_fermionic_op()\n",
+    "hamiltonian_ferm.display_format = \"sparse\"\n",
+    "hamiltonian_ferm.set_truncation(0)\n",
+    "print(hamiltonian_ferm)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93b2ca42-9947-45da-ad1e-7c11735dc5a5",
+   "metadata": {},
+   "source": [
+    "## Diagonalization and state preparation\n",
+    "A quadratic Hamiltonian can always be rewritten in the form\n",
+    "$$\n",
+    "    H = \\sum_{j} \\varepsilon_j b^\\dagger_j b_j + \\text{constant}\n",
+    "$$\n",
+    "where $\\varepsilon_0 \\leq \\varepsilon_1 \\leq \\cdots \\leq \\varepsilon_N$ are non-negative real numbers called orbitals energies and\n",
+    "the $\\{b_j^\\dagger\\}$ are a new set of fermionic creation operators\n",
+    "that also satisfy the canonical anticommutation relations.\n",
+    "These new creation operators are linear combinations of the original creation and annihilation operators:\n",
+    "$$\n",
+    "    \\begin{pmatrix}\n",
+    "    b^\\dagger_1 \\\\\n",
+    "    \\vdots \\\\\n",
+    "    b^\\dagger_N \\\\\n",
+    "    \\end{pmatrix}\n",
+    "    = W\n",
+    "    \\begin{pmatrix}\n",
+    "    a^\\dagger_1 \\\\\n",
+    "    \\vdots \\\\\n",
+    "    a^\\dagger_N \\\\\n",
+    "    a_1 \\\\\n",
+    "    \\vdots \\\\\n",
+    "    a_N\n",
+    "    \\end{pmatrix},\n",
+    "$$\n",
+    "where $W$ is an $N \\times 2N$ matrix. Given a basis of eigenvectors of the Hamiltonian, each eigenvector is labeled by a subset of $\\{0, \\ldots, N - 1\\}$, which we call the occupied orbitals. The corresponding eigenvalue is the sum of the corresponding values of $\\varepsilon_j$, plus the constant."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "62be7f36-3abe-4bde-a2cf-82a5c667fa27",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Shape of matrix W:\n",
+      "(4, 8)\n",
+      "\n",
+      "Orbital energies:\n",
+      "[0.29826763 4.38883678 5.5513683  5.64193745]\n",
+      "\n",
+      "Transformed constant:\n",
+      "-1.9402050758492795\n"
+     ]
+    }
+   ],
+   "source": [
+    "# get the transformation matrix W and orbital energies {epsilon_j}\n",
+    "(\n",
+    "    transformation_matrix,\n",
+    "    orbital_energies,\n",
+    "    transformed_constant,\n",
+    ") = hamiltonian.diagonalizing_bogoliubov_transform()\n",
+    "\n",
+    "print(\"Shape of matrix W:\")\n",
+    "print(transformation_matrix.shape)\n",
+    "print()\n",
+    "print(\"Orbital energies:\")\n",
+    "print(orbital_energies)\n",
+    "print()\n",
+    "print(\"Transformed constant:\")\n",
+    "print(transformed_constant)"
```

Comment:
This is a choice I guess to have the output over multiple lines.

Would it be easier on the eyes though and to read in the output keeping the text and value on one line by doing things more like this?
```
print(f"Shape of matrix W: {transformation_matrix.shape}")
print(f"Orbital energies: {orbital_energies}")
print(f"Transformed constant: {transformed_constant}")
```
Comment would apply to other places where printing is similar.

Reply:
Done.